src/controllers/plugins/Plugin.cpp

```diff
@@ -270,5 +273,19 @@ bool Plugin::hasFSPermissionFor(bool write, const QString &path)
     return false;
 }
 
+bool Plugin::hasHTTPPermissionFor(const QUrl &url)
```

Comment:
Why do we accept a url here?

Reply:
For cases like restricting protocols or urls a plugin can use. This is used to only allow http and https now